backend/cms/models/page.py

```diff
@@ -75,21 +86,37 @@ class PageTranslation(models.Model):
     """
 
     page = models.ForeignKey(Page, related_name='page_translations', on_delete=models.CASCADE)
-    permalink = models.CharField(max_length=60)
+    slug = models.SlugField(max_length=200, unique=True)
     STATUS = (
-        ('draft', 'Entwurf'),
-        ('in-review', 'Ausstehender Review'),
-        ('reviewed', 'Review abgeschlossen'),
+        ('draft', _('Draft')),
+        ('in-review', _('Pending Review')),
+        ('reviewed', _('Finished Review')),
     )
     title = models.CharField(max_length=250)
     status = models.CharField(max_length=9, choices=STATUS, default='draft')
     text = models.TextField()
-    language = models.ForeignKey(Language, on_delete=models.CASCADE)
+    language = models.ForeignKey(Language, related_name='pages', on_delete=models.CASCADE)
+    source_language = models.ForeignKey(
```

Comment:
Was it on purpose to add the language tree but keep the source_language property?

Reply:
No sorry, this doesn't make sense anymore. Thanks for pointing that out.